tests/sys/psa_crypto_mac/Makefile

```diff
@@ -1,12 +1,21 @@
 include ../Makefile.sys_common
 
+USEMODULE += embunit
```

Comment:
```suggestion
```

I would prefer to keep your tests consistent with the existing ones and not use embUnit for them (simply replace `TEST_ASSERT()` with the construction used in other tests).